src/p5.dance.js

```diff
@@ -826,6 +837,16 @@ module.exports = class DanceParty {
     }
   }
 
+  adjustSpriteDepth_(sprite) {
+    if (!this.spriteExists_(sprite)) {
+      return;
+    }
+
+    // Bias scale heavily (especially since it largely hovers around 1.0) but use
+    // Y coordinate as the first tie-breaker and X coordinate as the second.
+    sprite.depth = sprite.scale * 100 + sprite.y / 400 + sprite.x / 40000;
```

Comment:
It took me a minute to understand why 400 and 40000 here.  I wonder if we want to separate the weight and normalization for readability.
```js
sprite.depth =
  10000 * sprite.scale +
  100 * sprite.y / 400 +
  1 * sprite.x / 400; 
```

Reply:
Nice!